bridge-ext/src/main/java/com/liferay/faces/bridge/ext/event/internal/LiferaySharedPageTop.java

```diff
@@ -33,7 +33,7 @@
 
 /**
  * This class provides the ability to parse the {@link StringBundler} value of the LIFERAY_SHARED_PAGE_TOP request
- * attribute and remove duplicate CSS and/or JavaScript resources that are meant to be rendered in the <head>...</head>
+ * attribute and remove duplicate CSS and/or JavaScript resources that are meant to be rendered in the 'head'
```

Comment:
One place I used the &lt; for a fix but not for this <head> issue.  I was inconsistent.  Thanks for catching this.